essay4.md

```diff
@@ -1,5 +1,36 @@
 ## camerad Module Overview
 
+### cameras Directory
+Inside the [cameras directory](https://github.com/commaai/openpilot/tree/master/system/camerad/cameras), there are 5 main files, each with its key elements. 
+All of them have the general purpose of getting and parsing data from the camera 
+and configuring the camera.
+
+Camera common ([header file](https://github.com/commaai/openpilot/blob/master/system/camerad/cameras/camera_common.h)) contains functions common to using the camera. 
```

Comment:
I like the format of going through file by file with a summary at the start. I think the links are great too.